src/test/java/picard/pedigree/SexTest.java

```diff
@@ -0,0 +1,54 @@
+package picard.pedigree;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import picard.PicardException;
+
+import java.util.Arrays;
+
+public class SexTest {
+    @DataProvider(name="testSexDataProviderInt")
+    public Object[][] testSexDataProviderInt() {
+        return new Object[][] {
+                new Object[]{Sex.Male, 1, true},
```

Comment:
operator `new` not needed here
```suggestion
               {Sex.Male, 1, true},
```